styles/style.css

```diff
@@ -18,17 +18,14 @@ p, li {
 
 html {
   background-color: #00539F;
-  height: 100%;
 }
 
 body {
   width: 600px;
   margin: 0 auto;
   background-color: #FF9500;
-  padding: 0 20px;
-  height: 100%;
-  border-left: 5px solid black;
-  border-right: 5px solid black;
+  padding: 0 20px 20px 20px;
+  border: 5px solid black;
```

Comment:
This is a change I submitted in 2014. No idea how it never got merged and has ended up here ;-) I also don't know how to remove it.